layers/+lang/clojure/README.org

```diff
@@ -332,6 +332,25 @@ As this state works the same for all files, the documentation is in global
 
 *** Refactoring
 
+The following refactoring key bindings are enabled by default in clojure-mode:
+
+| Key Binding   | Description                                                    |
+|---------------+----------------------------------------------------------------|
+| ~SPC m r c i~ | cycle between if and if-not forms                              |
+| ~SPC m r c p~ | cycle privacy of defn and def forms                            |
+| ~SPC m r c (~ | convert coll to list                                           |
+| ~SPC m r c '~ | convert coll to quoted list                                    |
+| ~SPC m r c {~ | convert coll to map                                            |
+| ~SPC m r c #~ | convert coll to set                                            |
+| ~SPC m r c [~ | convert coll to vector                                         |
+| ~SPC m r t f~ | rewrite the following form to use the -> (thread first) macro. |
+| ~SPC m r t l~ | rewrite the following form to use the ->> (thread last) macro. |
+| ~SPC m r t h~ | thread another form into the surrounding thread                |
+| ~SPC m r u a~ | unwind all surrounding threading macros                        |
+| ~SPC m r u w~ | unwind threading macro one step at a time                      |
+
+The following refactorings require cljr-refactor to be [[https://github.com/syl20bnr/spacemacs/blob/develop/layers/%2Blang/clojure/README.org#enabling-sayid-or-clj-refactor][enabled]] and generally depend on a connected CIDER session.
```

Comment:
We should not have hardcoded URLs here since they will need manual intervention when releasing to master. It's enough to say `enabled` without linking since the instructions for this are in the same file.